src/components/Common/StatusBadgeMini/index.tsx

```diff
@@ -0,0 +1,52 @@
+import {
+  BellIcon,
+  CheckIcon,
+  ClockIcon,
+  MinusSmIcon,
+} from '@heroicons/react/solid';
+import { MediaStatus } from '@server/constants/media';
+
+interface StatusBadgeMiniProps {
+  status?: MediaStatus;
```

Comment:
status should probably not be an optional prop. We can then change the switch to get rid of the default and just write explicit cases for each status type.